frontend/rpm/template.go

```diff
@@ -317,6 +325,22 @@ func (w *specWrapper) PreUn() fmt.Stringer {
 	return b
 }
 
+func systemdPostScript(unitName string, cfg dalec.SystemdUnitConfig) string {
+	verb := "disable"
```

Comment:
This should take into account the default value and do nothing.

Reply:
Actually thinking about it we should probably only ever enable and do nothing when false.
I wrote up that other PR to make it a `*bool` with the idea that we have these preset files but I'm thinking maybe we don't need that?

basically

```go

if !cfg.Enable {
  return ""
}

// systemctl enable
```

Reply:
Yeah that seems reasonable. Because I believe `systemctl disable` is essentially a no-op unless the service is already enabled, right?

Reply:
And also potentially something we don't want to do willy-nilly.

Reply:
So this should be fixed. It does actually complicate things a bit though, because if we aren't explicitly disabling units, then we don't necessarily need a `%post` section or `%post` systemd dependency if no enabled units have been specified in the spec. I've added some logic to account for this